lua/dbee/ui/editor/init.lua

```diff
@@ -67,29 +64,6 @@ function EditorUI:new(handler, result, opts)
   return o
 end
 
--- Look for existing namespaces and their notes on disk.
----@private
-function EditorUI:search_existing_namespaces()
-  -- search all directories (namespaces) and their notes
-  for _, dir in pairs(vim.split(vim.fn.glob(self.directory .. "/*"), "\n")) do
-    if vim.fn.isdirectory(dir) == 1 then
-      for _, file in pairs(vim.split(vim.fn.glob(dir .. "/*"), "\n")) do
-        if vim.fn.filereadable(file) == 1 then
-          local namespace = vim.fs.basename(dir)
-          local id = file .. utils.random_string()
-
-          self.notes[namespace] = self.notes[namespace] or {}
-          self.notes[namespace][id] = {
-            id = id,
-            name = vim.fs.basename(file),
-            file = file,
-          }
-        end
-      end
-    end
-  end
```

Comment:
The bug here is that, this only loads notes that are "one level deep". i.e. consider this directory structure:
```
global/
├─ foo.sql
├─ bar.sql
file_source_/
├─ Users/
│  ├─ toh995/
│  │  ├─ .local/
│  │  │  ├─ state/
│  │  │  │  ├─ nvim/
│  │  │  │  │  ├─ dbee/
│  │  │  │  │  │  ├─ persistence.json0jdWVxQRAP/
│  │  │  │  │  │  │  ├─ foo.sql
│  │  │  │  │  │  │  ├─ bar.sql
```
The algorithm would only detect the notes `global/foo.sq` and `global/bar.sql`. Any other note that exists in the deeper levels will be missed in the algorithm.